model_name: "crazyflie_model_Narrow-Short"
GUI: true
controller: "GTC"
ceiling_height: 3.0
